suite-native/state/package.json

```diff
@@ -11,7 +11,7 @@
     },
     "dependencies": {
         "@reduxjs/toolkit": "1.9.5",
-        "@sentry/react-native": "6.5.0",
+        "@sentry/react-native": "^6.9.0",
```

Comment:
We need to wait until version 7 comes out because v6 is not compatible with v9 core.